astra-sim/system/collective/Algorithm.hh

```diff
@@ -32,6 +32,7 @@ class Algorithm : public Callable {
   uint64_t final_data_size;
   ComType comType;
   bool enabled;
+  static const int32_t TAG_OFFSET = 500000000;
```

Comment:
s/TAG_OFFSET/COLLECTIVE_COMM_TAG_OFFSET/g
Also, add comments either 1) pointing to the PR or 2) copy/pasting the PR description here